src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -92,7 +92,12 @@
     "DESCRIPTIONS": [
       "At the start of your next turn, you will Exhume some cards."
     ]
-  },
+  },"stsjorbsmod:ExhumeOnTurnXPower": {
+  "NAME": "Exhume",
+  "DESCRIPTIONS": [
+    "At the start of turn 3, you will Exhume a card."
```

Comment:
Would probably be better to use placeholders here (ideally for both the turn number and the card name), something like `At the start of turn #b%1$s, Exhume %2$s."